### ISSUE TITLE
Rrules don't have the correct start

### ISSUE THREAD

**Issue by PhilippJust** [iammatis/iKalendar#32]
## Expectation:

GIVEN I deserialize the following event
```
      BEGIN:VEVENT
      DTSTAMP:20200101T090030Z
      UID:test-id
      SUMMARY:Test Event 2
      DTSTART:20120201T103000Z
      RRULE:BYDAY=MO,FR;FREQ=WEEKLY
      DTEND;TZID=UTC:20120101T113000
      END:VEVENT
```
AND I get the RRULE of the event
THEN the RRULE has the option `dtstart` with content 20120201T103000Z

## Actual
... THEN the RRULE has the option `dtstart` with content `now`

**Comment by iammatis** [comment]
Yeah currently I'm using the `RRule` purely for `RRULE` property, so the start is not initialized, which could cause trouble if somebody initialized it with one, because then the event would have two `DTSTART` properties (I think). Same goes for parsing, it only parses the string `'RRULE:BYDAY=MO,FR;FREQ=WEEKLY'`, so it doesn't know about the `DTSTART` property. I'll try to fix it, thanks for bringing it up!
